src/reason_pprint_ast.ml

```diff
@@ -2597,9 +2597,21 @@ class printer  ()= object(self:'self)
         | (_::_, None) -> add_bar nameOf (normalize args)
         | (_::_, Some res) -> add_bar nameOf (normalize (args@[res]))
       in
+      (*
+        Attach attributes sitting on the entire variant leaf.
+          Example:
+            [ | `utf8 | `my_name [@bs.as "ascii"]]
+            type variantType = | Foo int [@onInt]
+        We don't want [@bs.as "ascii"], nor [@onInt] dropped. Because of [@bs.as "ascii"]
+        coming in from the Ast nodes outside of the Parsetree.constructor_declaration, while
```

Comment:
Not sure if the comment is true: I think both of the attributes are inside of contructor_declaration.  The difference is that in the first example, the type is an abstract type and all Rtags are defined in the manifest section of constructor_declaration. In the second example, the attribute [@onInt]  is decorating a variant type, which is also defined in constructor_delcaration.